### ISSUE TITLE
Dropdown:  'Error message' is not associated with the 'Error message example drop down'

### ISSUE THREAD

**Issue by cliffkoh** [OfficeDev/office-ui-fabric-react#10600]
Standalone tracking issue for item 3 in #6326.

**Comment by 999-Dev** [comment]
[Accessibility][Programmatic access]: 'Error message' is not associated with the 'Error message example drop down'
We observe that UI has been changed because there is no "error message example dropdown".
**Build Details:**
OS: Version 1903(OS Build 19005.1)
Browser: Edge
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Hence closing the bug.
